docs/A11y-Testing-Cross-Device.md

```diff
@@ -0,0 +1,11 @@
+# Testing on Multiple Devices for an [A11y](https://www.a11yproject.com/about/#what-does-the-term-a11y-mean) Swarm
+When developing any feature for this project we ensure we check our site as as accessible to as many users as we can by completing an [a11y swarm](https://bbc.github.io/accessibility-news-and-you/guides/accessibility-swarms.html). When conducting an A11y swarm, we often want to test a branch of Simorgh across multiple devices, e.g. on an A11y laptop with specialist software installed. In this document we specify some instructions to allow you to test cross device with Simorgh; we assume the dev machine is MacOS, any device can be used for a11y testing. The instructions allow you to load a locally running instance of Simorgh on your local wifi network.
+
+1. In Simorgh's route directory navigate to [`webpack.config.client.js`](https://github.com/bbc/simorgh/blob/latest/webpack.config.client.js)
+2. Replace `localhost` on [line 33](https://github.com/bbc/simorgh/blob/65743560d6721eef69ae64cc66d6b569cfd2d000/webpack.config.client.js#L33) and [line 40](https://github.com/bbc/simorgh/blob/65743560d6721eef69ae64cc66d6b569cfd2d000/webpack.config.client.js#L40) to `0.0.0.0`
```

Comment:
Are these two steps strictly necessary? I remember just having to update [this](https://github.com/bbc/simorgh/blob/latest/envConfig/local.env#L3) line, to match your machine's IP address e.g.  `http://192.168.0.xx:7080`

Reply:
Not entirely sure tbh, I followed them from @rhenshaw56 's original issue, I'll try and remember to give this a go without these steps next time I do an a11y swarm.